src/client/components/ContentCard/DeliveryInformation/DeliveryInformation.stories.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+import { DeliveryInformation } from './DeliveryInformation.component.js';
+
+export default { title: 'Components / Content Card' };
+
+export const Component = () => <DeliveryInformation />;
```

Comment:
It will be great if you add examples where you add values to the props. 
Also, there are 2 versions of how the information is distributed: one where everything is in one column, and the other version where the content is distributed in 2 columns
<img width="883" alt="140642026-d7919d5f-a10d-46a1-88f3-e3522523ad7f" src="https://user-images.githubusercontent.com/36211640/142497539-b6617897-d1f2-4b76-9bc9-0280b5a6964b.png">
<img width="1029" alt="140642023-95b98de1-b049-4e0b-88e6-249e41e82324" src="https://user-images.githubusercontent.com/36211640/142497545-ccc1cd64-1198-4b43-ac19-99032ba6306a.png">
.